frontend/src/v4/modules/viewerGui/viewerGui.selectors.ts

```diff
@@ -16,7 +16,7 @@
  */
 
 import { createSelector } from 'reselect';
```

Comment:
### Transformations are not resetting when you change view

Model I used is here: https://issue-4642.dev.3drepo.io/v5/viewer/DevTeamWorkspace/72d935e5-d15f-4bc4-8433-fb6d79d57301/bc8f82ef-db39-4cb0-a4f9-9bb47aa14dc9

#### Scenario 1
1. go into the sequence to a point where the object is transformed
2. Exit the sequence, transform resets (this is what we want, this is good)
3. perform 1 again
4. save a ticket with this view
5. exit sequence - The transform should reset (just like colour overrides) but it doesn't)

The action of exiting a sequence should have the same effect as changing a view (without changing the camera) -  object visibility/colour overrides/transforms should reset.

![23-12-15 05-18-05](https://github.com/3drepo/3drepo.io/assets/11945337/b53f460b-9010-43b6-8cf2-fdaf9cdc42d7)


#### Scenario 2
1. save a view
2. go into the sequence to a point where the object is transformed
3. save a ticket view
4. refresh 
5. go into the ticket with the transformed view
6. click on the saved view that should reset the transform

The transform doesn't reset - The transform should always reset when you're applying a new view.
![23-12-15 05-20-45](https://github.com/3drepo/3drepo.io/assets/11945337/98cc7e5c-5022-44ec-9b48-e1847728fc33)

Reply:
Still a bit odd:

#### transform is moving when I exit the ticket
- open a ticket with a transform
- open the sequence to a place that moves an object to a different place
- Exit the ticket

##### Current Behaviour:
It's moving the transformed object to where the custom ticket depicted it to be

##### Expected Behaviour:
Exiting a ticket detail should not be doing anything to the view (It doesn't change camera/coloured & hidden groups, so it should not reset transform)

#### scenario 2 is still broken
>#### Scenario 2
>1. save a view
>2. go into the sequence to a point where the object is transformed
>3. save a ticket view
>4. refresh 
>5. go into the ticket with the transformed view
>6. click on the saved view that should reset the transform
>
>The transform doesn't reset - The transform should always reset when you're applying a new view.
![23-12-15 05-20-45](https://github.com/3drepo/3drepo.io/assets/11945337/98cc7e5c-5022-44ec-9b48-e1847728fc33)

##### Current Behaviour:
It now resets the transform, but the colour overrides still exist
![24-01-02 03-43-47](https://github.com/3drepo/3drepo.io/assets/11945337/529536f8-9b34-46b3-b16f-86c8720e765e)

##### Expected Behaviour:
Should reset colour overrides just like when you're swapping between view 1 and view 2